microk8s-resources/actions/metrics-server.yaml

```diff
@@ -209,28 +152,15 @@ subjects:
   name: metrics-server
   namespace: kube-system
 ---
-kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: system:aggregated-metrics-reader
-  labels:
-    rbac.authorization.k8s.io/aggregate-to-view: "true"
-    rbac.authorization.k8s.io/aggregate-to-edit: "true"
-    rbac.authorization.k8s.io/aggregate-to-admin: "true"
-rules:
-- apiGroups: ["metrics.k8s.io"]
-  resources: ["pods", "nodes"]
-  verbs: ["get", "list", "watch"]
----
-apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: microk8s-admin
-roleRef:
+subjects:
+- kind: User
+  name: 127.0.0.1
   apiGroup: rbac.authorization.k8s.io
+roleRef:
   kind: ClusterRole
-  name: admin
-subjects:
-- apiGroup: rbac.authorization.k8s.io
-  kind: User
-  name: front-proxy-client
```

Comment:
I think we need to take into account the work on front proxy https://github.com/ubuntu/microk8s/pull/581

Reply:
Done